docker-compose-prod.yml

```
@@ -1,16 +1,23 @@
+  api-server:
+    image: sailin1367/napzak-api-prod:latest
+    container_name: napzak-api-prod
+    ports: ["8080:8080"]
     depends_on:
-      redis:
-        condition: service_healthy
-    networks:
-      - napzak-prod-net
+      redis: { condition: service_healthy }
+      rabbitmq: { condition: service_healthy }
+    networks : [napzak-prod-net]
```

YAMLlint (1.37.1)

[warning] too many spaces before colon

(colons)